File: Python3/API_v2_0/requestUsers.py
import requests


header={'Authorization': 'Bearer <YOUR-API-KEY>'}

# Get the entries as json from the server
res = requests.get('https://easyverein.com/api/stable/member/?showCount=true', headers=header)

# count the entries
count = res.json()
print(count["count"])

File: PHP/requestUsers.php
<?php

# Set the API version
$apiVersion = 'stable'

# Creating the header with the Token in it
$opts = [
    "http" => [
        "method" => "GET",
        "header" => "Authorization: Bearer <YOUR-API-KEY>"
    ]
];
$context = stream_context_create($opts);
# Execute the request
$content = file_get_contents('https://easyverein.com/api/' . $apiVersion . '/member/', false, $context);

# Decode the JSON and get the count value from it
$json = json_decode($content);
echo $json->count;

?>
